191.js

```diff
@@ -0,0 +1,16 @@
+//https://code.mu/ru/javascript/book/prime/inbuilt/string/searching/
+let str191 = 'https://code.mu/ru/javascript/book/prime/inbuilt/string/searching/'
+let flag = str191.startsWith('http://')
+if(flag){
+    console.log('yes')
+} else {
+    console.log('no')
+}
+
+let str1912 = 'goga.html'
+let flag1912 = str1912.endsWith('.html')
+if(flag1912){
+    console.log('yes')
+} else {
+    console.log('no')
+}
```

Comment:
навошта ствараеш дадатковые пераменую?
ты можаж адразу так
```suggestion
let str191 = 'https://code.mu/ru/javascript/book/prime/inbuilt/string/searching/'

if (str191.startsWith('http://')) {
    console.log('yes')
} else {
    console.log('no')
}

let str1912 = 'goga.html'
if (str1912.endsWith('.html')) {
    console.log('yes')
} else {
    console.log('no')
}
```

Reply:
паправіў